diff --git a/cyflann/flann_info.py b/cyflann/flann_info.py
index 16e4f4a..9ab4216 100644
--- a/cyflann/flann_info.py
+++ b/cyflann/flann_info.py
@@ -84,9 +84,13 @@ def get_flann_info(flann_dir=None, use_cache=True):
 
     if flann_dir:
         pre = partial(os.path.join, flann_dir)
-        lib_dirs = [pre('lib')]
-        if os.path.isdir(pre('lib64')) and sys.maxsize > 2**32:
-            lib_dirs.append(pre('lib64'))
+        if os.name == 'nt':
+            # flann.dll lives in bin/
+            lib_dirs = [pre('bin')]
+        else:
+            lib_dirs = [pre('lib')]
+            if os.path.isdir(pre('lib64')) and sys.maxsize > 2**32:
+                lib_dirs.append(pre('lib64'))
 
         _flann_info = {
             'libraries': ['flann', 'flann_cpp'],
